data/scripts/actions/items/die.lua

```diff
@@ -0,0 +1,36 @@
+local die = Action()
+
+function die.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	local position = item:getPosition()
+	local value = math.random(1, 6)
+	local isInGhostMode = player:isInGhostMode()
+
+	if not isInGhostMode then
+		position:sendMagicEffect(CONST_ME_CRAPS)
+
+		local spectators = Game.getSpectators(position, false, true, 3, 3)
+		for _, spectator in ipairs(spectators) do
+			player:say(player:getName() .. " rolled a " .. value .. ".", TALKTYPE_MONSTER_SAY, isInGhostMode, spectator, position)
+		end
+	end
+
+	local rolledSixCount = player:kv():get("die-rolled-six") or 0
+	if value == 6 then
+		player:kv():set("die-rolled-six", rolledSixCount + 1)
+
+		if rolledSixCount == 2 then
+			player:addAchievement("Number of the Beast")
+		end
+	else
+		player:kv():remove("die-rolled-six")
+	end
+
+	item:transform(5792 + value)
```

Comment:
it's 5791, because the ids are: from 5792 until 5797. not 5793 until 5798.
so: 5791 +1 or +2 or +3 
```suggestion
	item:transform(5791 + value)
```